### ISSUE TITLE
selection of default value (value_obj_id) by fk_provider and fk_dataowner in qgis version 3.24.11 not visible

### ISSUE THREAD

**Issue by tomep** [teksi/wastewater#480]
**Describe the bug**
After selecting, for example, the fk_provider, a selection appears in the "value_obj_id" field, but it is not visible.

**To Reproduce**
Exact steps to reproduce the behavior:
1. Go to Table "default values" and open the attributtable
2. put into editing mode
3. create a new entry
4. select in the first field e.g. fk_owner
5. nun möchte ich im zweiten attribut die organisation auswählen. diese ist jedoch nicht sichtbar.

**Expected behavior**
I can select the organization

**Screenshots / data**
![image](https://github.com/user-attachments/assets/1a1f4557-f456-4877-877f-27e578c62c74)

**Desktop (please complete the following information):**
 - TWW 2024.0.3
 - QGIS 3.24.11
 - OS win10

**Additional context**
In TWW 2024.0.1 and QGIS 3.24.9 it works

**Comment by cymed** [comment]
Have you loaded the organisations? Also I strongly encourage using the latest LTR if possible (3.34)

**Comment by tomep** [comment]
Yes. Organisations are importet. QGIS Version is 3.34.11

**Comment by ponceta** [comment]
Something seems not working 100% on QGIS side with the attribute table and entity creation.

Can you show us what you see when changing to form view :

<img width="640" alt="image" src="https://github.com/user-attachments/assets/3b3305d7-26c5-4bc7-a0d2-445847a75c35">

**Comment by tomep** [comment]
OK, here i see a warning:

***@***.***

**Comment by ponceta** [comment]
Yes, the warning is here to tell you that the value_obj_id has to be filled. But can you select the desired organisation through that window? (You will be able to select an organisation only if you imported / created these before)

![image](https://github.com/user-attachments/assets/7515dcd7-a49d-4e5f-a0b7-12ae204288be)

If this works, then it is mainly a documentation / UX/UI issue -> we already defined that we wanted to be able to have a specific GUI for this setting but the tool is missing right now.

**Comment by ponceta** [comment]
@cymed we could maybe reword `value_obj_id` with something more explicit like `fk_organisation` ?

**Comment by cymed** [comment]
The functionality allows all kind of obj_id entries, not only organisations. Thus the name

**Comment by tomep** [comment]
ok, sorry it works. i can choose empty fields ;-).
![image](https://github.com/user-attachments/assets/1d53f670-15e4-4d2a-8102-06eed0e2177e)
but i have 7 empty rows to choes.
in the organisation table i have also 7 organisations
![image](https://github.com/user-attachments/assets/8c2b40cb-92e6-4a2f-8285-c0a398790867)
if I select one of them, I can save
![image](https://github.com/user-attachments/assets/a2883488-099d-4eb6-90a0-bb909966c77f)

**Comment by ponceta** [comment]
Theses are not empty rows, it seems like your project has translated a value that should not have been translated.

If you can show us what widget configuration for value_obj_id in your `default_values` layer properties ?

Here is a valid configuration but translation to _de could have altered it.

![image](https://github.com/user-attachments/assets/45b312e4-04e5-4ee0-b63c-a9503025356c)

**Comment by ponceta** [comment]
This is mainly a translation issue because we don't have it on the base project.

**Comment by tomep** [comment]
the layer properties look like this for me:

![Image](https://github.com/user-attachments/assets/28a5c0e1-1cbc-4210-af90-1fe1bf17784a)

when I now select identifier it becomes visible in the selection

![Image](https://github.com/user-attachments/assets/62699631-8165-4a10-9b82-15a03427f4f9)

**Comment by urskaufmann** [comment]
is changed now in transifex

**Comment by urskaufmann** [comment]
all text "identifier" with context 
fk_dataowner\:valuerelationvalue or 
fk_provider\:valuerelationvalue or
fk_owner\:valuerelationvalue or
fk_operator\:valuerelationvalue
should NOT be translated to "Bezeichnung" (because the field in od_organisation, that is used in the value relation, is "identifier" and is not translated)

Is now reset in transifex to "identifier" . 
@ponceta : pleace create new *_de.qm-file

**Comment by urskaufmann** [comment]
I used the newest translations- file here  https://github.com/teksi/wastewater/actions/runs/12922392617
and it works: all organisations are now ok.
Can this new project-translations file be put into the latest release download?

**Comment by ponceta** [comment]
Release 2024.0.5 is up to date with the proposed file :

https://github.com/teksi/wastewater/releases/tag/2024.0.5

To @3nids we shoud block the generation of QGIS strings of project \:valuerelationvalue since it breaks QGIS functionnality when translating.

Issue is opened QGIS side https://github.com/qgis/QGIS/issues/60255

**Comment by ponceta** [comment]
French translation is fixed too. Add a comment as reference to this issue on the first item on Transifex.

**Comment by 3nids** [comment]
we'll look into this at the same time than value maps.

**Comment by urskaufmann** [comment]
Also other valuerelationvalue with text identifier and translation-text Bezeichnung make problems (e.g. vw_tww_reach.fk_pipe_profil: there are no names of the pipe profiles). I changed the translation in 24 cases to identifier and created now a new project translation. Works now.

Blocking all valuerelationvalue:  will this also block the "value_en" in the valuerelations to the valuelists? translation to value_de is ok and works good. This translation should not be blocked

**Comment by sjib** [comment]
With project file rc6 I get problems in setting the default_values: see https://github.com/orgs/teksi/discussions/125#discussioncomment-14564873

A problem of TMMT or the project?

**Comment by ponceta** [comment]
When you start with TWW using TMMT, you will have no organisation at first if you don't have data in you database.

If you import some organisations, they will not be set to `tww_active` = true at first because we hope people will load the full VSA organisation which is providing a large amount organisation.

When you will import you data, related organisations will automatically set to active and you will be able to set these to default values.

💡This has to be documented somewhere :)
